### PR TITLE
data binding

### PR DESCRIPTION
**Pull request by connievi**

### CODE REVIEW

File: app/src/main/java/com/example/recipeopedia/RecipeAdapter.java [hunk 1]
```diff
@@ -34,7 +36,9 @@ public RecipeAdapter(List<Recipe> recipes) {
     public ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType)
     {
         Log.d(TAG, "onCreateViewHolder");
-        View recipeView = LayoutInflater.from(parent.getContext()).inflate(R.layout.item_recipe, parent, false);
+        Context context = parent.getContext();
```
Comment (JayArri) [inline app/src/main/java/com/example/recipeopedia/RecipeAdapter.java]: I think this is also an instance where we don't want to save any references to context. @ericacooksey correct me if I'm wrong

File: app/src/main/java/com/example/recipeopedia/fragments/RecipeListFragment.java [hunk 2]
```diff
@@ -49,15 +49,17 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
...
         recipeAdapter = new RecipeAdapter(mRecipes);
         rvRecipes.setAdapter(recipeAdapter);
         rvRecipes.setLayoutManager(new LinearLayoutManager(getContext()));
 
+        queryRecipes("chicken"); // keyword is CHICKEN to not overload API with so many requests
+
         EditText etSearch = view.findViewById(R.id.etSearch);
-        etSearch.addTextChangedListener(new TextWatcher() {
+        // TODO: uncomment below for final product
+        /*etSearch.addTextChangedListener(new TextWatcher() {
```
Comment (JayArri) [inline app/src/main/java/com/example/recipeopedia/fragments/RecipeListFragment.java:62]:
```suggestion
        /*etSearch.addTextChangedListener(new TextWatcher()) {
```

Comment (JayArri) [reply app/src/main/java/com/example/recipeopedia/fragments/RecipeListFragment.java:62]: I know this is commented out for now, but I noticed you're missing a closing bracket

File: app/src/main/java/com/example/recipeopedia/RecipeAdapter.java (hunk 1 above)
Comment (ericacooksey) [reply app/src/main/java/com/example/recipeopedia/RecipeAdapter.java]: yep, what @JayArri said :)

Comment (ericacooksey) [reply app/src/main/java/com/example/recipeopedia/RecipeAdapter.java]: oh wait actually i think this is fine, it's just a locally-scoped variable

File: app/src/main/java/com/example/recipeopedia/fragments/RecipeListFragment.java (hunk 2 above)
Comment (connievi) [reply app/src/main/java/com/example/recipeopedia/fragments/RecipeListFragment.java:62]: it's an anonymous function so i don't need the closing bracket there!

**Pull request merged**